src/components/Departments/DepartmentsComponent.jsx

```diff
@@ -0,0 +1,138 @@
+
+import NavBarComponent from '../../_components/NavBarContainer';
+import LoaderComponent from '../LoaderComponent';
+import Cards from '../common/Card/Card';
+import ItemsNotFoundComponent from '../common/ItemsNotFoundComponent';
+import Paginator from '../common/PaginationComponent';
+import StatusMessageComponent from '../common/StatusComponent';
+import PageHeader from '../common/PageHeader';
+import DepartmentModal from '../../_components/Departments/DepartmentModalContainer';
+
+class DepartmentsComponent extends React.Component {
+  state = {
+    limit: 10,
+    activePage: 1,
+    modalOpen: false,
+    editModalOpen: false,
+    location: {}
+  };
+
+  componentDidMount() {
+    this.props.loadDepartments(this.state.activePage, this.state.limit);
+  }
+
+  handleRowChange = (e, data) => {
+    this.setState({ limit: data.value });
+    this.props.loadDepartments(this.state.activePage, data.value);
+  };
+
+  handlePaginationChange = (e, { activePage }) => {
+    this.setState({ activePage });
+    this.props.loadDepartments(activePage, this.state.limit);
+  };
+
+  handleToggleModal = () => {
+    this.setState({ modalOpen: !this.state.modalOpen });
+    this.props.resetMessage();
+  };
+
+  handleEditToggleModal = (data = {}) => {
+    this.setState({
+      editModalOpen: !this.state.editModalOpen,
+      location: data
+    });
+    this.props.resetMessage();
+  };
+
+  getTotalPages = () => Math.ceil(this.props.departmentsCount / this.state.limit);
+
+  render() {
+    const { isLoading, departmentsList, error, resetMessage, entity } = this.props;
+    const hasDepartments = !isEmpty(departmentsList);
+    const showStatus = error;
+    const showAction = entity === 'andela-departments';
+    const showNotFound = !isLoading && !hasDepartments && !showStatus;
+
+    return (
+      <NavBarComponent>
+        <div className="assets-list">
+          <PageHeader header="Andela Departments">
+            <div className="header-modal-button">
+              <DepartmentModal
+                showTrigger
+                title="Add Department"
+                mode="add"
+                onToggle={this.handleToggleModal}
+                open={this.state.modalOpen}
+              />
+            </div>
+          </PageHeader>
+
+          <DepartmentModal
+            title="Update An Andela Department"
+            mode="edit"
+            onToggle={this.handleEditToggleModal}
+            open={this.state.editModalOpen}
+            data={this.state.location}
+            key={`edit-${this.state.location.id || 1}`}
+          />
+
+          {showStatus && (
+            <StatusMessageComponent
+              message={error}
+              className="error-status"
+              reset={resetMessage}
+            />
+          )}
+
+          {isLoading && !showStatus && <LoaderComponent />}
+
+          {!isLoading && hasDepartments && (
+            <Cards
+              data={departmentsList}
+              headings={['name', 'id']}
```

Comment:
Why do we have the `id` on the heading?

Reply:
it is the number (id) displayed on the card below the department name

Reply:
Yes, I was asking if there's a reason to display the id below the name.